### PR TITLE
Luke/mgmt service docker compose

### PR DESCRIPTION
**Pull request by Luke-Rogerson**
- [Expose management service status page](https://github.com/orbs-network/v3-node-setup/commit/72182c22f1c025e4d1b7f6ff31467a0d74494860)
- [Update install script to use sudo and work better with EC2](https://github.com/orbs-network/v3-node-setup/commit/eaaaeed68fa37e3a05c68b7022514855a12c1602)
- [Update WORKDIR to match EC2 ubuntu](https://github.com/orbs-network/v3-node-setup/commit/6bc3341e28770fcb5dc41be9211982b41f5b95a3)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
 FROM ubuntu:22.10
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install -y software-properties-common python3 python3-pip
+# Add sudo to make more like EC2 instance
+RUN apt-get update && apt-get install -y software-properties-common python3 python3-pip sudo locales
 
-WORKDIR /home
+# EC2 instances usually have locale settings
+RUN locale-gen en_US.UTF-8 && update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
```
Comment (uv-orbs) [inline Dockerfile:10]: en_UK next time :)

File: Dockerfile [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+  LC_ALL=en_US.UTF-8
 
 # Needed to allow crons to run in the container
 RUN echo "#!/bin/sh\nexit 0" > /usr/sbin/policy-rc.d
 
-COPY setup setup
-COPY manager manager
-COPY deployment deployment
+# Use non-root user (Docker by default uses root)
+RUN useradd -ms /bin/bash ubuntu && \
```
Comment (uv-orbs) [inline Dockerfile:19]: Is this the same thing is done by EC2?

File: deployment/docker-compose.yml [hunk 3]
```diff
@@ -7,18 +7,21 @@ services:
     ports:
       - 80:80
     volumes:
-      - ./nginx.conf:/etc/nginx/conf.d/default.conf
+      - $HOME/deployment/nginx.conf:/etc/nginx/conf.d/default.conf
```
Comment (uv-orbs) [inline deployment/docker-compose.yml:10]: why cant we keep it as its been fore? because of ubuntu user? I would rather keep the yaml as simple as possible

**Review by uv-orbs — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (Luke-Rogerson) [reply Dockerfile:19]: Yeah, EC2 uses an `ubuntu` user

File: Dockerfile (hunk 1 above)
Comment (Luke-Rogerson) [reply Dockerfile:10]: I think en_US is more universal?

**Pull request merged**